package.json

```diff
@@ -234,6 +234,10 @@
   "dependencies": {
     "@elastic/datemath": "^5.0.3",
     "@elastic/eui": "34.6.0",
+    "@redis-ui/components": "^38.1.4",
```

Comment:
`Section` component in `@redis-ui/components` v38.7.1 comes with a bug fix that we need for the **Vector Search** page -> **Manage Indexes** drawer -> Index info boxes

When the box section is collapsed, it should display summary information about the index. However, there was a bug in Redis UI that makes this information not appear once the component is rendered inside a Drawer. They resolved the issue on their end, and we have to update Redis UI dependencies.